addons/pos_hr/models/hr_employee_public.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class HrEmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    def read(self, fields):
```

Comment:
```suggestion
    def read(self, fields=None, load='_classic_read'):
```
This is causing issues in SAAS and SH at the moment when `load` is passed here: https://github.com/odoo/odoo/blob/16.0/addons/hr/models/hr_employee.py#L236